sh/ci_after_script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -e
+
+SH_DIR=$(readlink -f $(dirname "$0"))
+source "${SH_DIR}/logging.sh"
+WPTDASHBOARD_DIR=${WPTDASHBOARD_DIR:-$(readlink -f "${SH_DIR}/..")}
+
+cd "${WPTDASHBOARD_DIR}"
+
+info "Stopping and removing docker instance 'wptdashboard-instance'"
+docker stop wptdashboard-instance && docker rm wptdashboard-instance
```

Comment:
Is stopping and deleting the instance necessary? The container should presumably stop after the test run, and if it's being run in CI the image management should presumably be taken care of by the runner.

Reply:
This is mostly for completeness. I don't think I tried leaving this out; I would presume that Travis won't hang on the running image (because the "script" script will run to completion).